client/src/Components/Navigation.js

```diff
@@ -0,0 +1,26 @@
+import React, {useState, useEffect} from 'react';
+import { Link } from 'react-router-dom';
+import '../App.css';
+
+
+function Navigation (props) {
+    const [click, setClick] = useState (false);
+
+    return (
+        <nav className = 'navbar'>
+
+        <div id = 'navContainer'>
+        <Link to = '/main'> Main </Link>
+        <Link to = '/tutoring'> Tutoring </Link>
+        <Link to = '/cats'> Hobbies  </Link>
```

Comment:
Try to be consistent with the names! Why is Hobbies, linking to a page of `/cats`